### ISSUE TITLE
Focus order accessibility improvements

### ISSUE THREAD

**Issue by trebuan202** [gchq/stroom#5405]
2.4.3 Focus Order (Partially supports)

When items are selected using the keyboard ( e.g. security, user groups ), focus should be given to that viewable item. Currently, in Stroom, when you select an item ( e.g. security, user groups ) focus remains on the main menu. Also, when inside the viewable item ( e.g. security, user groups ) focus order using the keyboard keys does not follow a sequential ( left-to-right, top-down ) approach.
